### PR TITLE
test json conf with webpack

### PR DESCRIPTION
**Pull request by Swiip**
@zckrs can you look at this.
Pro: it's js code, so much easy to debug and test
Con: generated conf file is poorly formated

### CODE REVIEW

**Comment by Swiip** [comment]
Well, I geeked a little and proposed something better formated which reduce the cons ;)

**Comment by zckrs** [comment]
Remove generators/app/templates/conf/webpack.conf.js.save :p

File: generators/app/index.js [hunk 1]
```diff
@@ -39,11 +39,49 @@ module.exports = fountain.Base.extend({
...
+        conf.entry = [
+          'webpack/hot/dev-server',
+          'webpack-hot-middleware/client',
+          lit`\`./\${path.join(conf.paths.src, 'index')}\``
+        ];
+      } else {
+        conf.entry = lit`\`./\${path.join(conf.paths.src, 'index')}\``;
+      }
+
+      if (this.props.framework === 'react') {
```
Comment (zckrs) [inline generators/app/index.js]: Same if at line 59. May be combine.

File: generators/app/index.js [hunk 2]
```diff
@@ -39,11 +39,49 @@ module.exports = fountain.Base.extend({
     },
 
     conf: function () {
-      this.fs.copyTpl(
-        this.templatePath('conf'),
-        this.destinationPath('conf'),
-        { framework: this.props.framework }
-      );
+      const lit = this.lit;
```
Comment (zckrs) [inline generators/app/index.js]: NOTE: lit is define here https://github.com/FountainJS/fountain-generator/blob/proto-json-conf/lib/file-utils.js#L39

File: generators/app/conf.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+const lit = require('fountain-generator').lit;
```
Comment (zckrs) [inline generators/app/conf.js:1]: :100:

**Pull request merged**